src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CryptoHelpers.cs

```diff
@@ -22,7 +22,9 @@ name switch
                 "urn:mpeg:mpeg21:2003:01-REL-R-NS:licenseTransform" => new XmlLicenseTransform(),
                 "http://www.w3.org/2000/09/xmldsig# X509Data" => new KeyInfoX509Data(),
                 "http://www.w3.org/2000/09/xmldsig# KeyName" => new KeyInfoName(),
+#pragma warning disable CA1416 // This call site is reachable on all platforms. 'DSAKeyValue' is unsupported on: 'ios', 'maccatalyst', 'tvos'
```

Comment:
Started an offline with @buyaa-n @GrabYourPitchforks and @bartonjs to figure out what to do about this and the other exclusion in SignedXmlDebugLog.cs (change below).